Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/ContentStreamForking/ForkContentStreamWithoutDimensions.feature

```diff
@@ -70,7 +70,7 @@ Feature: ForkContentStream Without Dimensions
 
     And the event NodePropertiesWereSet was published with payload:
       | Key                          | Value                                                   |
-      | workspaceName                | "user"                                                  |
+      | workspaceName                | "user-test"                                             |
```

Comment:
Why was this test affected?

Reply:
I think it was wrong the whole time, but the test did't fail.